logger/lumberjack_const_test.go

```diff
@@ -102,55 +96,11 @@ func TestWriteToTruncatedFile(t *testing.T) {
 	assert.Equal(t, f[19:], []byte("Z I! SHOULD BE FIRST\n"))
 }
 
-func TestWriteToFileInRotation(t *testing.T) {
```

Comment:
Why remove this test? I'm assuming it's related to a change in lumberjack during the upgrade but what's the consequence?

Reply:
Because with the new change that register lumberjack logger inside Telegraf, we can no longer create the logger like it does in this test 
```
lumberjackLogger := &lumberjack.Logger{
		Filename:   config.Logfile,
		MaxSize:    maxsize,
		MaxBackups: 5,
		MaxAge:     1,
		Compress:   false,
	}
```
Also I restored NewTelegrafWriter() to private as it was in Telegraf, we shouldn't be change it to public and use it, influxdb is not maintaining it as a public method, and we use it anyway which result in it breaks us in this upgrade. So with all those changes, it's hard to test this here properly. I did manually verify the rotation works as it was before, and I start to question the value of this unit test, it's not like any part of the lubmerjack log rotation could be changed by us, so what unit feature is this test covering?